Watchman.Discord/Areas/Statistics/Models/MessageInformation.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using Watchman.Integrations.MongoDB;
+
+namespace Watchman.Discord.Areas.Statistics.Models
+{
+    public class MessageInformation : Entity //todo stworzyć warstwe z encjami na które będziemy mapować nasze obiekty
+    {
+        //możliwe że powinniśmy trzymać w obiektach niżej mniej informacji
+        //zaoszczędzilibyśmy na mocy obliczeniowej, a informacje mogłyby być później uzupełniane na podstawie matchowania
+        //w bazie trzymalibyśmy osobno listy z wszystkimi autorami, kanaałami i serwerami które obsługujemy
+        //w planach jest żeby bot obsługiwał więcej serwerów niż devscord
```

Comment:
Na zasadzie przechowywania jedynie ID autora, kanału, serwera itp., zamiast całego obiektu?

Reply:
tak
byłyby wtedy osobne obiekty trzymające większą ilość informacji
służące do generowania raportów itp 

chociaż z drugiej strony trzymanie tych informacji też ma znaczenie, bo zapisujemy kto kiedy zmienił swoją nazwę 
to nie jest bardzo istotna informacja, ale warto trzymać to na przyszłość
na razie te dane nie ważą dużo